content/api/cypress-api/custom-commands.md

```diff
@@ -21,6 +21,8 @@ evaluated via an import statement in your
 ```javascript
 Cypress.Commands.add(name, callbackFn)
 Cypress.Commands.add(name, options, callbackFn)
+Cypress.Commands.addAll(callbackArgs)
```

Comment:
Instead of `callbackArgs` can we call this `callbackObj`?

Reply:
[`877d1b1` (#4414)](https://github.com/cypress-io/cypress-documentation/pull/4414/commits/877d1b1d47056cab0375bf50c0d4d4481703b660)